src/test/scala/Test.scala

```diff
@@ -24,12 +31,15 @@ s"""|============================
      }
 
      val pdf = {
-       val renderer = new PdfRenderer(classLoader)
+       val renderer = new PdfRenderer(classLoader, rendererWithFonts)
        renderer.toBytes(body)
      }
 
      Files write (`test.pdf`, pdf)
 
      todo
+
+    // Todo Write proper tests please
```

Comment:
You are right, its so much work for what's worth. But I was thinking more to test more the elements that we expect to be there on a document.